### ISSUE TITLE
The radio won't play

### ISSUE THREAD

**Issue by lukapaunovic** [uktechreviews/AIY#1]
Radio doesn't play, but code seems fine :(
When I cancel the script, THEN it starts playing and I have to stop it with
mpc clear

What could be the reason?

**Comment by lukapaunovic** [comment]
The problem was running it with OS python, so i ran it with one included with AIY

/home/pi/AIY-voice-kit-python/env/bin/python /home/pi/AIY-voice-kit-python/src/my_assistant.py

Then it works.
one question, when the radio starts playing, the only way to stop it is by button?

**Comment by uktechreviews** [comment]
You should be able to us OK Google and then "Stop My Radio" to stop the radio without having to press the button.  I have found though if the volume of the radio is loud the mic can't pick it up.  This is why I added the button function,

**Comment by lukapaunovic** [comment]
Yeah, you are right .... the speaker is too close to the microphone.
When i set mpc volume to 30 then it's ok :)
I added commands for volume up and volume down :)

**Comment by rmele09** [comment]
@lukapaunovic I'm having the same problem and I i don't understand your solution. The radio only plays after I quit the script. Could you please explain how you fixed this? Thanks!

**Comment by lukapaunovic** [comment]
Have you placed the .py file in src folder?

You need to run it using Python from the AIY folder, not the one from OS.

`/home/pi/AIY-voice-kit-python/env/bin/python3 -u /home/pi/AIY-voice-kit-python/src/my_assistant.py`
